packages/backend/discovery/_templates/opstack/L2OutputOracle/template.jsonc

```diff
@@ -17,22 +17,22 @@
   "fields": {
     "proposer": {
       "target": {
-        "role": "Proposer"
+        "permissions": [{ "type": "propose", "delay": 0 }]
```

Comment:
Let's remove `"delay": 0` in all places. This just adds unnecessary noise. We also don't add it in other places.